.gitignore

```diff
@@ -17,4 +17,6 @@ out.txt
 /builds/
 /dist/
 
-/.vscode/
\ No newline at end of file
+/.vscode/
+
+unit_test*.txt
```

Comment:
In your tests create temporary files so that the test cleans up after itself.

Reply:
Added clean up logic